### PR TITLE
feat(api)!: API enchancement

### PR DESCRIPTION
**Pull request by samiralavi**
This PR is a for the renaming in the API to rename `alias` to `eid` and `id` to `uuid` for experiments.

### CODE REVIEW

**Review by jatinriverlane — APPROVED** [review]
Thanks @samiralavi

File: aqueductcore/frontend/src/components/organisms/ExperimentDetailsActionButtons/index.tsx [hunk 1]
```diff
@@ -180,6 +180,7 @@ function ExperimentDetailsActionButtons({ isEditable, isDeletable, experimentDet
                         }
                         sx={{ minWidth: 140, ml: 1 }}
                         onClick={ToggleAddToFavourites}
+                        title="toggle-favourite"
```
Comment (str-anger) [inline aqueductcore/frontend/src/components/organisms/ExperimentDetailsActionButtons/index.tsx:183]: What is this? =)

File: extensions/python-example/solve_alpha.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import os
```
Comment (str-anger) [inline extensions/python-example/solve_alpha.py:1]: Extensions are deleted and moved to `external/extensions` as a submodule

**Review by str-anger — CHANGES_REQUESTED** [review]
A minor change requested -- you've added an extension file, probably accidentaly.

File: aqueductcore/frontend/src/components/organisms/ExperimentDetailsActionButtons/index.tsx (hunk 1 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/components/organisms/ExperimentDetailsActionButtons/index.tsx:183]: It's a title which is used in tests

**Review by safeamiiir — APPROVED** [review]

File: extensions/python-example/solve_alpha.py (hunk 2 above)
Comment (samiralavi) [reply extensions/python-example/solve_alpha.py]: I have removed it.

**Review by str-anger — APPROVED** [review]

**Pull request merged**